.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,34 @@
+name: Python package
+
+on: [push]
```

Comment:
Please specify the target branches like below.
```
on:
  push:
    branches:
      - master
    tags:
      - v*
  pull_request:
    branches:
      - master
```

Reply:
will do , thanks.